### ISSUE TITLE
The accuracy of resnet family

### ISSUE THREAD

**Issue by swift1988** [VITA-Group/Nasty-Teacher#13]
Hi, thanks for your excellent work~
Training  a baseline model(train_scratch.py) on cifar100,  resnet18 achieved a higher accuracy than resnet34 and resnrt50.  I have nothing changed.
Do you know what is wrong? Looking for your reply!

**Comment by HowieMa** [comment]
Hi, thanks for asking. Could you please show the specific results you get, rather than just say "higher"? 

I didn't try Resnet34 on CIFAR-100, but I did try ResNet18 and 50 on it as shown in my paper. 
With the hyperparameters in the folder (https://github.com/VITA-Group/Nasty-Teacher/tree/main/experiments/CIFAR100), 
we have 77.44% for resnet18 and 78.12% for resnet50 on CIFAR-100, where ResNet50 performs slightly better than ResNet18. As you can see, the difference is just 0.68%. 

You may expect to see slightly different results due to different seeds, different versions of PyTorch, NumPy, etc. Based on my experience, the STD on CIFAR can be ~0.5%. To this end, it is possible that you obtain a slightly better result on Resnet18.

**Comment by swift1988** [comment]
Thx for your reply.
I have 78.085% for resnet18 and 76.681% for resnet50 on CIFAR-100.  The version of PyTorch and Numpy is 1.4.0 and 1.17.2, which is identical with your envs as reported in requirments.txt.

Here are training logs for resnet18 and resnet50.

[training-50.txt](https://github.com/VITA-Group/Nasty-Teacher/files/8468979/training-50.txt)
[training-18.txt](https://github.com/VITA-Group/Nasty-Teacher/files/8468980/training-18.txt)

**Comment by HowieMa** [comment]
I guess that it is because of the random seed ... 

My training log at that time and trained resnet50 are here: https://drive.google.com/file/d/1YrdQOLubB9i5JJb9bezrgAJV2MDr0ILC/view?usp=sharing. 
Hope it is helpful.

**Comment by swift1988** [comment]
Ok then, I will try that.
thx for sharing.
